src/Yolo.java

```diff
@@ -27,19 +26,22 @@
 
 	public static void main(String[] args) throws InterruptedException {
 		System.load("D:\\OpenCV\\opencv\\build\\java\\x64\\opencv_java430.dll");
-		VideoCapture cap = new VideoCapture(); // Load video using the videocapture OpenCV class//
-		cap.open(filePath);
 		String modelWeights = "D:\\OpenCV\\yolov3-tiny.weights";
 		String modelConfiguration = "D:\\OpenCV\\yolov3-tiny.cfg";
+		
+		
+		VideoCapture cap = new VideoCapture();
+		cap.open(filePath);
```

Comment:
Do we need to release the resources allocated by the open call?

Reply:
Based on the OpenCV Javadocs, VideoCapture does not have a method for closing the stream. I believe it automatically stops it once the video file/source is ended.